Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -23,10 +23,15 @@ import SwiftSyntax
 % for node in [SyntaxBuildableNode(syntax_node) for syntax_node in SYNTAX_NODES if syntax_node.is_buildable()]:
 %   type = node.type()
 %   base_type = node.base_type()
+%   conformances = [base_type.buildable(), type.expressible_as()]
+%   has_with_trailing_comma_trait = node.node.traits and 'WithTrailingComma' in node.node.traits
```

Comment:
I didn’t check in detail (and might be wrong) but could this be simplified to `type.has_with_trailing_comma_trait()`? In which case you could also eliminate the `has_with_trailing_comma_trait` variable.

Reply:
`has_with_trailing_comma_trait()` looks for a node related to the type in `SYNTAX_NODES`. Since we already have the `node` here, I think it is more optimal to use it right away. But if we don't care about those scripts performance, then yes, I can simplify it